ons_alpha/static_src/sass/components/design_system_overrides/_back-to-top.scss

```diff
@@ -0,0 +1,11 @@
+@use 'config' as *;
+
+// overrides to ons back-to-top styles
```

Comment:
I experimented with wrapping the back-top-top link in a custom div that we could apply our own styling to, without overriding the design system styles. However, this wasn't possible as the js to make the link sticky when scrolling relies on the content being the previous element: https://github.com/ONSdigital/design-system/blob/1300be4d8f0f58c2a32b80a3bea85a8fd6ee82d6/src/components/back-to-top/back-to-top.js#L4